README.adoc

```diff
@@ -124,13 +124,13 @@ These values can replace or amend the default configuration for the `recordIssue
 See https://github.com/jenkinsci/warnings-ng-plugin/blob/master/doc/Documentation.md#configuration[Warnings NG Plugin documentation]
 for a list of available configuration parameters.
 * `timeout`: (default: `60`) - the number of minutes for build timeout, cannot be bigger than 180, i.e. 3 hours.
+* `noIncrementals`: (default: `false`) - de-activates incremental version publication (by default generating a Git version based will be attempted).
```

Comment:
> (by default generating a Git version based will be attempted)

this sentence doesn't make sense to me, I think it could be just dropped.

```suggestion
* `noIncrementals`: (default: `false`) - de-activates incremental version publication.
```

Reply:
Thanks, I've merged your suggestion, now build is failing running the spotless check goal with an obscure reason, I don't have this error when building locally, any clue ?